_component_design/toast.md

```diff
@@ -5,40 +5,136 @@ layout: component
 category: Components
 usage: >
   Toasts are a specialized notification component used to provide feedback on the status of an actions taking place in the background. Toasts are meant to be ephemeral and capture users attention by having the highest Z-index of all components.
-preview-image: components/preview-images/toast.svg
-status: deprecated
-resource: false
-last-modified: 2017-08-17
+preview-image: preview-images/toast.svg
+status: stable
+resource: true
+last-modified: 2017-11-05
 ---
 
 {% include toc.html %}
 
+## Introductions
+
 <div class="hxRow">
-{% column left:"hxCol" %}
-{% figure [caption:"Default"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/toast.svg){:width="100%"}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- A toast notification provides information for action outside of the user’s current context. User understand that clicking on any links within the toast notification will take them out of the current context. 
+- This pattern should NOT be used for action in context to the page/ region. In that case, consider an alert bar. 
+- A toast notification is **not** used for promotional or marketing material; such as the announcement of new features on the current page.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"The toast notification component"] [class:"image bg-light border"] %}
+![]({{site.baseurl}}/assets/images/components/notifications/toasts/toast-hero.svg)
 {% endfigure %}
 {% endcolumn %}
-{% column left:"hxCol" %}
-{% figure [caption:"Alert"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/toast-alert.svg){:width="100%"}
+
+</div>
+
+<div class="hxRow">
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+### Best Practices
+
+- Keep messages short and succinct, but also helpful. In the case of an error, if there is an action the user can take (correct the errors on the form or submit the form again), use a call to action (CTA) if possible. 
+- Use complete sentences. If only 1 sentence is used, do not include a period. However, if there are 2+ sentences, use periods.
+- Try not to exceed more than 4 lines of your message (excluding the CTA).
+- Only use one CTA per toast. Never include “Dismiss” or “Close” as a CTA. 
+- Clicking on the “X” closes the toast notification. Hovering over it keeps it open until the cursor moves off of it. (See “Timing” section for more detail.)
```

Comment:
add link for timing section.
Also, s/`(See “Timing” section for more detail.)`/`(See Timing for more details.)` (with `Timing` linking out to the proper section)

Reply:
Use markup tags instead of quotes.

Reply:
fixed.